Build/Customizations/Properties.props

```diff
@@ -2,15 +2,12 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <!-- Special settings for re-mix: -->
-    <AssemblyInfoFileForMixins>$(SolutionDirectory)Remotion\Mixins\AssemblyInfoShared.cs</AssemblyInfoFileForMixins>
```

Comment:
we still have the remotion AssemblyInfoShared file on the disk we should remember to delete it some time. ticket please :)